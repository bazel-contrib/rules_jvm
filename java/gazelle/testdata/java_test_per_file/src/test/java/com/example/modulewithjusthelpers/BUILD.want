load("@rules_java//java:defs.bzl", "java_library")

java_library(
    name = "modulewithjusthelpers",
    testonly = True,
    srcs = [
        "Helper.java",
        "withdirectory/Helper.java",
    ],
    _gazelle_imports = ["com.google.common.math.IntMath"],
    _java_packages = [
        "_TESTONLY!com.example.modulewithjusthelpers",
        "_TESTONLY!com.example.modulewithjusthelpers.withdirectory",
    ],
)
