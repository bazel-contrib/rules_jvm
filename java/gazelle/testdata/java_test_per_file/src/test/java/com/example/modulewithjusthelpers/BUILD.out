load("@rules_java//java:defs.bzl", "java_library")

# gazelle:java_module_granularity module

java_library(
    name = "modulewithjusthelpers",
    testonly = True,
    srcs = [
        "Helper.java",
        "withdirectory/Helper.java",
    ],
    deps = ["@maven//:com_google_guava_guava"],
)
